plugins/gradle/gradle.plugin.sh

```diff
@@ -0,0 +1,24 @@
+#! bash oh-my-bash.module
+
+function gradle-or-gradlew() {
+  # find project root
+  # taken from https://github.com/gradle/gradle-completion
+  local dir="$PWD" project_root="$PWD"
+  while [[ "$dir" != / ]]; do
+    if [[ -x "$dir/gradlew" ]]; then
+      project_root="$dir"
+      break
+    fi
+    dir="${dir:h}"
+  done
+
+  # if gradlew found, run it instead of gradle
+  if [[ -f "$project_root/gradlew" ]]; then
+    echo "executing gradlew instead of gradle"
```

Comment:
In case the user pipes the commands, I think this type of informative message should go to stderr. What do you think?

```suggestion
    echo "executing gradlew instead of gradle" >&2
```